docs/basics.md

```diff
@@ -238,7 +238,7 @@ Here is an example of backends and servers definition:
 
 # Launch
```

Comment:
I think we should rename this part to Configuration.
Could you add a section at the beginning about the static / dynamic configuration?